kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/setupConfigurationsAndCompilations.kt

```diff
@@ -7,7 +7,9 @@ import org.gradle.api.Project
 import org.gradle.api.tasks.Exec
```

Comment:
IMO this file is getting out of hand.
We should separate it into multiple parts as best as possible asap
We can do this in a separate PR if you want but IMO we definitely have to now.

Reply:
I agree on this one.
But I think this should be in a specific PR. I'd like we can add easily configuration to this plugin so that adding a platform is easy.